src/Main.java

```diff
@@ -0,0 +1,38 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        //1.Initiate an Arraylist. Using For loop add integers from 1 to 10
+        List<Integer> test = new ArrayList<>(Arrays.asList());
+        for (int i = 1; i < 11; i++) {
+            test.add(i);
+        }
+        System.out.println(test);
+
+        //2.Create an ArrayList [1,2,3,6,7,3,2,1]. Remove all 2 from Arraylist
+        List<Integer> list1 = new ArrayList<>(Arrays.asList(1, 2, 3, 6, 7, 3, 2, 1));
+        for (int i = 0; i < list1.size(); i++) {
+            if (list1.get(i) == 2) {
+                list1.remove(i);
+            }
+        }
```

Comment:
Please use an iterator to remove items from the list or try to fix it, because if you delete items using your solution, you will get incorrect results (especially if you place an even number of twos one after the other)